src/app/components/search/LanguageFilter.js

```diff
@@ -0,0 +1,176 @@
+/* eslint-disable @calm/react-intl/missing-attribute */
```

Comment:
Please remove this eslint-disable directive

Reply:
Done